### ISSUE TITLE
Subscription field must return AsyncIterable

### ISSUE THREAD

**Issue by commandtab** [mirumee/ariadne#149]
When running ariadne 0.3.0 using the counter example in the [Subscriptions docs](https://github.com/mirumee/ariadne/blob/master/docs/subscriptions.rst), the server errors with `TypeError: Subscription field must return AsyncIterable. Received: None`. However, I'm returning an int in the `counter_resolver`, just as the example shows. What am I missing?

I've put together a complete example here:
https://github.com/commandtab/ariadne-subscriptions

Related: there are a number of `import` statements missing from that example that I had to hunt down myself. Once I wrap my head around this, I'd be happy to submit a PR to improve the documentation.

Thanks!

**Comment by rafalp** [comment]
Have you tried annotating your generator with `@subscription.source("counter")`? You need to explicitly tell Ariadne the source/generator of data for subscriptions fields.

> Related: there are a number of import statements missing from that example that I had to hunt down myself.

This if fair - we should keep our examples copy-pasteable for people experimenting with them themselves.

**Comment by commandtab** [comment]
Oh! That makes sense. That seemed to work — it no longer errors out when attempting to subscribe. 

Now I can successfully subscribe in the GraphQL Playground, though I'm not seeing the `counter` int responses 🤔Is the subscription `subscription { counter }` not correct? It does show "Listening..." in the right side view of the Playground.

**Comment by rafalp** [comment]
Hi, sorry for late response.

I've checked your code and its using `@subscription.field` instead of `@subscription.field('counter')` and errors with `NameError: name 'asyncio' is not defined` to the console. After fixing those two issues I've got it working:

<img width="1680" alt="Zrzut ekranu 2019-05-6 o 15 03 57" src="https://user-images.githubusercontent.com/750553/57227081-d1040500-7010-11e9-8e89-c9cf129984bf.png">

I'll revisit those docs in `0.4` and also include extra checks so we'll throw `ValueError` when dev misses required `("name")` after decorator.
